Core/src/org/sleuthkit/autopsy/logicalimager/configuration/EditNonFullPathsRulePanel.java

```diff
@@ -41,7 +41,7 @@
  * Edit non-full paths rule panel
  */
 @SuppressWarnings("PMD.SingularField") // UI widgets cause lots of false positives
-public class EditNonFullPathsRulePanel extends javax.swing.JPanel {
+class EditNonFullPathsRulePanel extends javax.swing.JPanel {
```

Comment:
If class is final it will fix issue with over-ridable method calls in the constructor.